apps/offlineevents/views.py

```diff
@@ -0,0 +1,120 @@
+from django.contrib import messages
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext_lazy as _
+from django.views import generic
+
+from adhocracy4.modules import views as module_views
+from adhocracy4.projects.models import Project
+from adhocracy4.rules import mixins as rules_mixins
+from apps.dashboard.mixins import DashboardBaseMixin
+
+from . import forms
+from . import models
+
+
+class OfflineEventDetailView(module_views.ItemDetailView):
+    model = models.OfflineEvent
+    permission_required = 'meinberlin_offlineevents.view_offlineevent'
+
+    @property
+    def project(self):
+        return self.get_object().project
+
+
+class OfflineEventMgmtView(DashboardBaseMixin,
```

Comment:
not sure if we should use DashboardBaseMixin here or write a more general mixin that does not already overwrite `get_permission_object` and `get_success_url` as those are very specific to the dashboard create/edit project views

also i think the `slug` kwarg is to lax for a general purpose mixin. i'd prefer project_slug